Neos.Fusion/Resources/Private/Fusion/Root.fusion

```diff
@@ -167,6 +167,65 @@ prototype(Neos.Fusion:ResourceUri) {
   @exceptionHandler = 'Neos\\Fusion\\Core\\ExceptionHandlers\\AbsorbingHandler'
 }
 
+# Renders a link pointing to a controller/action
+#
+# Usage:
+# link = Neos.Fusion:Link.Action {
+#     package = 'Some.Package'
+#     controller = 'Standard'
+#     action = 'index'
+# }
```

Comment:
This example would render an empty tag.
```suggestion
# link = Neos.Fusion:Link.Action {
#     package = 'Some.Package'
#     controller = 'Standard'
#     action = 'index'
#
#     content = 'Some Link'
# }
```

or

```suggestion
# link = afx`
    <Neos.Fusion:Link.Action package="Some.Package" controller="Standard" action="index">
        Some Link
    </Neos.Fusion:Link.Action>
```

would make more sense IMO